packages/hub/config/structure.sql

```diff
@@ -818,6 +818,22 @@ CREATE TABLE graphile_worker.migrations (
 
 ALTER TABLE graphile_worker.migrations OWNER TO postgres;
 
+--
+-- Name: account_nonces; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public.account_nonces (
+    id uuid NOT NULL,
+    account_address text NOT NULL,
```

Comment:
So we will have a table that only handles the nonce for crank address? And in that case, this table will only have one row because instead of inserting a new row when the nonce of the crank increases, we could update the value of the nonce column on the existing row.

My idea on this table is to have a nonce lock feature for any accounts, not only for the crank. And I'm not in the idea of having multiple crank addresses because this nonce lock feature is not aimed for that. To do that, we could have another table or mapping from this table to `scheduled_payment_attempts` table. But it's not necessary for now.

Reply:
the the idea of nonce locking is necessary for the EOA _sending_ the txn. and I believe the only EOA sending txn's is the crank. in terms of the safe address that the crank is executing txns for, i don't see how that is relevant at all to the nonce locking--it seems like superfluous data... if that is necessary to track, I suggest using a different table.

Reply:
yeah--that's a great name that explains the context for the table very well

Reply:
renamed